src/main/java/Greeting.java

```diff
@@ -0,0 +1,30 @@
+class Greeting extends Event {
+    int status;
+
+    public Greeting() {
+        super(false);
+        this.status = -1;
+    }
+
+    public Greeting(int inPlay) {
+        super(false);
+        this.status = inPlay;
+    }
+
+    public Event toNext() {
```

Comment:
Maybe can be more clearer on the nameing, like nextEvent or toNextEvent?
```suggestion
    public Event toNext() {
```